src/components/Proposal/ProposalContainer.tsx

```diff
@@ -108,18 +111,26 @@ class ProposalContainer extends React.Component<IProps, IState> {
       isVotingYes
     } = this.props;
 
-    const redeemable = currentAccountRedemptions && (
+    const beneficiaryHasRewards = currentAccountRedemptions && (
       currentAccountRedemptions.beneficiaryReputation ||
       currentAccountRedemptions.beneficiaryNativeToken ||
+      (currentAccountRedemptions.beneficiaryEth && dao.ethCount >= currentAccountRedemptions.beneficiaryEth)
```

Comment:
I think we always want to show the rewards and just indicate if there isn't enough ETH to actually carry out the reward

Reply:
We do, in [this](https://github.com/daostack/alchemy/pull/137/files/dab40ee32d0b023a076f8e76a03161c4b10663b5#diff-c02babe5ab2a47081fe5b019ee9f62f2R432) line. This boolean only indicates if there are rewards that the beneficiary can actually redeem.

If there are rewards that can't be redeemed, the button and tooltip will show up but will be disabled.

Reply:
right